crypto/src/main/kotlin/net/corda/v5/crypto/DigitalSignature.kt

```diff
@@ -36,4 +36,19 @@ open class DigitalSignature(
          */
         val context: Map<String, String>
```

Comment:
Why is the class open?

Reply:
The context is also really vague and either we need to say what you can expect in it, or remove it

Reply:
Done